services/server/src/server/apiv1/verification/private/stateless/private.stateless.handlers.ts

```diff
@@ -0,0 +1,392 @@
+      FROM compiled_contracts_sources
+      JOIN sources ON sources.source_hash = compiled_contracts_sources.source_hash
+      WHERE compiled_contracts_sources.compilation_id = $1`,
+      [verifiedContract.compilation_id],
+    );
+
+    // Initialize jsonInput with data from database
+    const compilerVersion = verifiedContract.version;
+
+    // Create sources object from the query result
+    const sources: Record<string, { content: string }> = {};
+    for (const source of sourcesResult.rows) {
+      sources[source.path] = { content: source.content };
+    }
+
+    // Create settings from compiler_settings JSON field
+    const settings = JSON.parse(verifiedContract.compiler_settings);
+
+    // Initialize jsonInput with the required fields
+    const jsonInput: SolidityJsonInput = {
+      language: "Solidity",
+      sources: sources,
+      settings: settings,
+    };
+
+    // Get the file path and contract name from fully_qualified_name
+    const metadataCompilationTarget = (
+      JSON.parse(verifiedContract.metadata) as Metadata
+    ).settings.compilationTarget;
+    const compilationTarget = {
+      name: Object.values(metadataCompilationTarget)[0],
+      path: Object.keys(metadataCompilationTarget)[0],
+    };
+
+    const compilationArtifacts = JSON.parse(
+      verifiedContract.compilation_artifacts,
+    );
+
+    const creationCodeArtifacts = JSON.parse(
+      verifiedContract.creation_code_artifacts,
+    );
+
+    const runtimeCodeArtifacts = JSON.parse(
+      verifiedContract.runtime_code_artifacts,
+    );
+
+    const compilation = new SolidityCompilation(
+      solc,
+      compilerVersion,
+      jsonInput,
+      compilationTarget,
+    );
+
+    compilation.compilerOutput = {
+      contracts: {
+        [compilationTarget.path]: {
+          [compilationTarget.name]: {
+            abi: compilationArtifacts.abi,
+            userdoc: compilationArtifacts.userdoc,
+            devdoc: compilationArtifacts.devdoc,
+            metadata: verifiedContract.metadata,
+            storageLayout: compilationArtifacts.storageLayout,
+            evm: {
+              bytecode: {
+                object: verifiedContract.creation_code,
+                sourceMap: creationCodeArtifacts.sourceMap,
+                linkReferences: creationCodeArtifacts.linkReferences,
+              },
+              deployedBytecode: {
+                object: verifiedContract.runtime_code,
+                sourceMap: runtimeCodeArtifacts.sourceMap,
+                linkReferences: runtimeCodeArtifacts.linkReferences,
+                immutableReferences: runtimeCodeArtifacts.immutableReferences,
+              },
+            },
+          },
+        },
+      },
+    };
+
+    compilation["compile"] = async () => {
+      // Override so that it doesn't compile
+    };
+
+    compilation["generateCborAuxdataPositions"] = async () => {
+      // Override so that it doesn't generate auxdata positions
```

Comment:
Don't we need to generate these in order to have the `matchBytecodes` function working?

Reply:
Ah I see you mock the properties below. Nevermind then :)